.github/workflows/runtime_build.yml

```diff
@@ -52,3 +52,45 @@ jobs:
           name: build_${{ matrix.worker_id }}_${{ matrix.release_channel }}
           path: |
             build
+
+  process_artifacts_combined:
+    name: Process artifacts combined
+    needs: build_and_lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 18.20.1
+          cache: yarn
+          cache-dependency-path: yarn.lock
+      - name: Restore cached node_modules
+        uses: actions/cache@v4
+        id: node_modules
+        with:
+          path: "**/node_modules"
+          key: ${{ runner.arch }}-${{ runner.os }}-modules-${{ hashFiles('yarn.lock') }}
+      - run: yarn install --frozen-lockfile
+      - name: Restore archived build
+        uses: actions/download-artifact@v4
+        with:
+          path: build
+          merge-multiple: true
+      - name: Display structure of build
+        run: ls -R build
+      - run: echo ${{ github.sha }} >> build/COMMIT_SHA
+      - name: Scrape warning messages
```

Comment:
Note: this used to be a separate job in circleci but it was really unnecessary to spin up a new CI worker just for this. So I merged this step into process artifacts since it's the only job that needs this.